akvo/rsr/spa/webpack.config.js

```diff
@@ -138,6 +138,11 @@ const config = () => {
 
     plugins: [
       new webpack.NamedModulesPlugin(),
+      new HtmlWebpackPlugin({
+        template: `${__dirname}/app/index.html`,
+        filename: 'index.html',
+        inject: 'body',
+      }),

```

Comment:
Why have you added this in the dev config?

Reply:
Because we want to use the index.html and inject the bundle config in the dev environment. Why should it be a problem to add it to dev config?